sdk/search/search/review/search.api.md

```diff
@@ -131,13 +130,13 @@ export class SearchIndexClient<T> {
     readonly apiVersion: string;
     autocomplete<Fields extends keyof T>(options: AutocompleteOptions<Fields>): Promise<AutocompleteResult>;
     count(options?: CountOptions): Promise<number>;
```

Comment:
Should be `countDocuments` so we know what's being counted. In the future, we may add another resource to `SearchIndexClient` besides "docs", so it would be good to disambiguate (except for `search`, `suggest`, and `autocomplete`, where it's obvious from the context and those names are well-known on their own).

Reply:
renamed